analyses/focal-cn-file-preparation/04-prepare-cn-file.R

```diff
@@ -240,9 +240,10 @@ cnv_no_xy <- cnv_df %>%
 # Merge and annotated no X&Y
 autosome_annotated_cn <- process_annotate_overlaps(cnv_df = cnv_no_xy,
                                                    txdb_exons = tx_exons) %>%
-  # mark possible amplifications in autosomes
+  # mark possible amplifications and deep loss in autosomes
   dplyr::mutate(status = dplyr::case_when(
     copy_number > (2 * ploidy) ~ "amplification",
+    copy_number == 0 ~ "deep loss",
```

Comment:
```suggestion
    copy_number == 0 ~ "deep deletion",
```

just so we don't have to change it later, updating to this standard nomenclature